namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-activities-management.tf

```diff
@@ -0,0 +1,119 @@
+resource "kubernetes_secret" "hmpps-activities-management" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = "activities-api-dev"
+  }
+
+  data = {
+    access_key_id     = module.hmpps-domain-events.access_key_id
+    secret_access_key = module.hmpps-domain-events.secret_access_key
+    topic_arn         = module.hmpps-domain-events.topic_arn
+  }
+}
+
+module "hmpps_activities_management_queue" {
```

Comment:
@timharrison-moj do we want to include lines 14 on-wards just worry about consuming messages as a first cut? Easy enough to split out for now.